src/Bundle/Config/GridConfig.php

```diff
@@ -28,10 +31,16 @@ public function addGrid(GridBuilderInterface $gridBuilder): GridConfigInterface
 
     public function toArray(): array
     {
-        $output = [];
+        if (count($this->grids) <= 0) {
+            return [];
+        }
+
+        $output = [
+            'grids' => [],
+        ];
```

Comment:
Is this intentional change, that previous empty result was returning `[]`, but now it will return `['grids' => []]` instead?

```suggestion
        $output = [];
```

Reply:
Maybe you did not notice I flip the first if condition to make the method return earlier.

But the behavior of this method remain the same.

Reply:
@diimpp your comment makes me fix the CI, I forgot to set the array key.